client-templates/nexus2/accept.json.sample

```diff
@@ -18,7 +18,7 @@
       "origin": "${BASE_NEXUS_URL}"
     },
     {
-      "method": "any",
+      "method": "GET",
```

Comment:
```
{
  "method": "GET",
  "path":"/internal/healthcheck",
  "origin":"${BASE_NEXUS_URL}"
}
```
and for unauthenticated access we can advise the following path instead: 
`"path":"/service/local/status"`
based on the plugin's REST API in Nexus 2.x, see - 
- https://help.sonatype.com/repomanager2/configuration/plugins-and-the-rest-api
- https://repository.sonatype.org/nexus-restlet1x-plugin/default/docs/path__status.html

Reply:
no need to add, from the same reason above.